src/v2/cookbook/practical-use-of-scoped-slots.md

```diff
@@ -70,17 +70,17 @@ export default {
 }
 ```
 
-This is just part of a working example, you can find the whole example in the Codesandbox below.
+Éste es simplemente un ejemplo funcional, usted puede encontrar el ejemplo completo abajo.
```

Comment:
Remove accent from `Éste`. The E should not have an accent because the word is a "llana" that ends in a vowel. It simply does not have an accent at all. 

We surely have other cases to correct in other documents but we can correct them as we detect them.